Classes/Domain/Service/NewsImportService.php

```diff
@@ -203,7 +203,7 @@ protected function hydrateNewsRecord(
                 }
 
                 // no file found skip processing of this item
-                if ($file === false) {
+                if ($file === false || $file === NULL) {
```

Comment:
Isn't it better to set $file in line 202 to `null` instead of `false`?
Then the condition could be only `if ($file === null) {` (null lowercase) and cover all negative cases, or do I mistake about it?

Reply:
Should work i presume ...

Reply:
Shorter Code. I will change it.